qcodes/instrument_drivers/Keysight/keysightb1500/constants.py

```diff
@@ -618,6 +618,42 @@ class MeasurementMode(IntEnum):
         Ls_D = 401
         Ls_Q = 402
 
+    class Name(StrEnum):
+        R = 'Resistance'
+        G = 'Conductance'
+        Z = 'Impedance'
+        Y = 'Admittance'
+        Cp = 'Parallel Capacitance'
+        Cs = 'Series Capacitance'
+        Lp = 'Parallel Inductance'
+        Ls = 'Series Inductance'
+        X = 'Reactance'
+        B = 'Susceptance'
+        THETA_RAD = 'Phase'
+        THETA_DEG = 'Phase'
+        D = 'Dissipation Factor'
+        Q = 'Quality Factor'
+        Rp = 'Parallel Resistance'
+        Rs = 'Series Resistance'
+
+    class Unit(StrEnum):
+        R = 'ohm'
+        G = 'S'
+        Z = 'ohm'
+        Y = 'S'
+        Cp = 'F'
+        Cs = 'F'
+        Lp = 'H'
+        Ls = 'H'
+        X = 'ohm'
+        B = 'S'
+        THETA_RAD = 'radian'
+        THETA_DEG = 'degree'
+        D = 'unit'
+        Q = 'unit'
+        Rp = 'ohm'
+        Rs = 'ohm'
```

Comment:
keep forgetting - are using "ohm" or "ohms" across qcodes, in other drivers?

Reply:
I saw some drivers using ohms. I will change it to ohms then. But did not understand why ohm(s) was used in other drivers?

Reply:
> why ohm(s) was used in other drivers

not sure i understand the question. some instruments do measure resistance or allow to set smth such as a resistance. wrt ohm vs ohms vs Ohm vs Ohms vs <big omega> - i don't know, qcodes doesn't dictate a standard for the units yet.